docs/cockroachdb.md

```diff
@@ -0,0 +1,391 @@
+  storageclass.storage.k8s.io/openebs-hostpath            openebs.io/local                                           Delete          WaitForFirstConsumer   false                  25m
+  storageclass.storage.k8s.io/openebs-jiva-default        openebs.io/provisioner-iscsi                               Delete          Immediate              false                  25m
+  storageclass.storage.k8s.io/openebs-snapshot-promoter   volumesnapshot.external-storage.k8s.io/snapshot-promoter   Delete          Immediate              false                  25m
+  storageclass.storage.k8s.io/premium-rwo                 pd.csi.storage.gke.io                                      Delete          WaitForFirstConsumer   true                   38m
+  storageclass.storage.k8s.io/standard (default)          kubernetes.io/gce-pd                                       Delete          Immediate              true                   38m
+  storageclass.storage.k8s.io/standard-rwo                pd.csi.storage.gke.io                                      Delete          WaitForFirstConsumer   true                   38m
+  ```
+
+#### Accessing CockroachDB
+
+- We will be using the built in sql-client for accessing and running some sql queries. Enter into one of the cockroachdb pod by using exec command.
+  ```bash
+  kubectl exec -it cockroachdb-2 -- ./cockroach sql --certs-dir cockroach-certs
+  ```
+  ```bash
+  #sample output
+  #
+  # Welcome to the CockroachDB SQL shell.
+  # All statements must be terminated by a semicolon.
+  # To exit, type: \q.
+  #
+  # Server version: CockroachDB CCL v20.2.5 (x86_64-unknown-linux-gnu, built 2021/02/16 12:52:58, go1.13.14) (same version as client)
+  # Cluster ID: e51bfde5-2e75-4991-844e-d769f4b9b684
+  #
+  # Enter \? for a brief introduction.
+  #
+  root@:26257/defaultdb>
+  ```
+- Run some basic SQL queries.
+  Example command:
+  ```bash
+  root@:26257/defaultdb> CREATE DATABASE bank;
+  root@:26257/defaultdb> CREATE TABLE bank.accounts (id INT PRIMARY KEY, balance DECIMAL);
+  root@:26257/defaultdb> INSERT INTO bank.accounts VALUES (1, 1000.50);
+  root@:26257/defaultdb> SELECT * FROM bank.accounts;
+    id | balance
+  -----+----------
+     1 | 1000.50
+  (1 row)
+  ```
+
+- Create database user with password for accessing the database using web UI.
+  ```bash
+  root@:26257/defaultdb> CREATE USER roach WITH PASSWORD 'Q7gc8rEdS';
+  root@:26257/defaultdb> GRANT admin TO roach;
+  ```
+ 
+- Create one more database, which will be used later for running benchmark load
+  ```bash
+  root@:26257/defaultdb> CREATE DATABASE sbtest;
+  root@:26257/defaultdb> \q
+  ```
+ 
+#### Accessing the database's web interface. 
+We will be using NodePort for accessing the service.
+In production environment either use loadbalancer or ingress services as per your requirement
+
+- Create a new node port service using the following
```

Comment:
Create a new node port service using the following > Create a new node port service using the following YAML spec. The following sample spec is saved as `cockroachdb-public-node-port.yaml`.